### ISSUE TITLE
`require-description` cannot have more than one exempt tag

### ISSUE THREAD

**Issue by maribethb** [gajus/eslint-plugin-jsdoc#975]
## Expected behavior
`require-description` takes an option `exemptedBy` that can be an array of tags. If any of these tags are present, the description block is not required.

## Actual behavior
If the `exemptedBy` option is an array with more than one tag, it seems that all of the tags will be ignored, and we will get an error: `Missing JSDoc block description` even if one of the exempt tags is present.

If the array contains only one tag, then it behaves as expected.

## ESLint Config

```js
module.exports = {
  root: true,

  parser: 'babel-eslint',

  parserOptions: {
    ecmaVersion: 6,
    sourceType: 'module',
  },

  plugins: [
    'jsdoc',
  ],

  rules: {
    'jsdoc/require-description': [
      'warn',
      {
        // Don't require descriptions if these tags are present.
        'exemptedBy': ['inheritdoc, param, return, returns, type'],
      },
    ],
  },
};
```

## ESLint sample

This shouldn't require a description because `returns` is one of the tags in our `exemptedBy` array. But we still see the `Missing JSDoc block description` warning.

```js
/**
 * @returns always true
 */
function test() {
  return true;
}
```

## Environment

- Node version:  `16.19.0`
- ESLint version  `v7.32.0`
- `eslint-plugin-jsdoc` version: `39.7.5` and `37.2.3`

**Comment by brettz9** [comment]
This line should be fixed to become an array of strings like this:

```js
'exemptedBy': ['inheritdoc', 'param', 'return', 'returns', 'type'],
```

It then works as intended. Closing as that should resolve, but feel free to comment further as needed.

**Comment by maribethb** [comment]
Thanks  🤦🏻‍♀️ That worked, can't believe I didn't notice this after how long I spent making sure it reproduced lol.
